.github/workflows/linters.yml

```diff
@@ -1,56 +0,0 @@
-name: Linters
-
-on: pull_request
-
-env:
-  FORCE_COLOR: 1
-
-jobs:
-  lighthouse:
-    name: Lighthouse
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "18.x"
-      - name: Setup Lighthouse
-        run: npm install -g @lhci/cli@0.7.x
-      - name: Lighthouse Report
-        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
-  webhint:
-    name: Webhint
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
```

Comment:
- [ ] As stated by the previous reviewer, kindly add this file content back. It seems you have it deleted on the repo, but it's present on your local machine. I've attached a screenshot of files in this branch you're working on to see that it's not present. The GitHub workflow file is very important to safely check your code and will be a major requirement going forward in future projects.

- Kindly add the file back with its content, and you're good to go👍
![image](https://user-images.githubusercontent.com/50990925/227386898-d2f7fc27-a066-4cd3-97e6-7b022dcf7333.png)

Reply:
@Shedrack-Sunday added the linters files to Branch, do the linters file only have be pushed to branch and not to the main? this is what I don't understand ! do these files have to be on one branch or in both? what is PR(primary or main branch)?